src/Doctrine/ODM/Repository/BaseDocumentRepository.php

```diff
@@ -72,11 +74,11 @@ public function findBy(array $criteria, array $orderBy = null, $limit = null, $o
     {
         $paginatorAdapter = new CallbackPaginatorAdapter(
             function () use ($criteria, $orderBy, $limit, $offset) {
-                /** @var \Doctrine\ODM\MongoDB\Cursor $results */
+                /** @var Iterator $results */
                 $results = $this->createSelectQuery($criteria, $orderBy, $limit, $offset)->execute();
 
-                // Return as array of documents (and not the cursor)
-                return $results->toArray(false);
+                // Return as array of documents (and not the iterator)
+                return $results->toArray();
```

Comment:
I'm not sure this change has not any side effects, because as far is I could tell by looking at the code, `toArray(false)` was not the default behavior.

Reply:
This caused https://github.com/detailnet/dfw-persistence-module/issues/23